ups/validate_base.table

```diff
@@ -0,0 +1,8 @@
+setupRequired(pyyaml)
+setupRequired(numpy)
+setupRequired(astropy)
+
+setupOptional(base)
```

Comment:
Correct. It only needs to work if other packages have been setup that need it. Those packages will have enabled `base` so the import will work.